src/index.ts

```diff
@@ -136,6 +136,25 @@ export function get(key: string): any {
   return initializedAdapter && getInstance(initializedAdapter).get(key);
 }
 
+/**
+ * Retrieves all values that correspond to a given list of keys.
+ *
+ * Example:
+ *  const a = 1;
+ *  const b = 2;
+ *  const sum = a + b;
+ *  store.set({a, b, sum})
+ *  const results = store.getByKeys(['a', 'b', 'sum']); // {a: 1, b: 2, sum: 3}
+ *
+ * @param {Array<string>} keys
+ * @returns {*}
+ */
+export function getByKeys(keys: string[]): any {
```

Comment:
You can make it a generic function to have a specific return type. 
```suggestion
export function getByKeys<T>(keys: string[]): T[] {
```